Headers/DebugServer2/Host/FileIO.h

```diff
@@ -0,0 +1,33 @@
+//
+// Copyright (c) 2014-present, Facebook, Inc.
+// All rights reserved.
+//
+// This source code is licensed under the University of Illinois/NCSA Open
+// Source License found in the LICENSE file in the root directory of this
+// source tree. An additional grant of patent rights can be found in the
+// PATENTS file in the same directory.
+//
+
+#ifndef __DebugServer2_Host_FileIO_h
+#define __DebugServer2_Host_FileIO_h
+
+#include "DebugServer2/Base.h"
+
+#if defined(OS_WIN32)
+#include "DebugServer2/Host/Windows/FileIO.h"
+#else
+#include "DebugServer2/Host/POSIX/FileIO.h"
+#endif
+
+namespace ds2 {
+namespace Host {
+
+#if defined(OS_WIN32)
+using Windows::FileIO;
```

Comment:
It seems like you're keeping the interface of `FileIO` identical between POSIX and Windows (which is what we should do). If that's the case, you don't need these three header files with these inclusions; you can just put the interface of `FileIO` in `Headers/DebugServer2/Host/FileIO.h` and use it everywhere.